rfcs/045-sierra-work-relationships/README.md

```diff
@@ -0,0 +1,377 @@
+
+These RFCs concern the linking of Works to Series, and of Works to other Works
+coming from Sierra.
+
+It is apparent that these are two different things, with two different 
+manifestations in the API and Website. Though they do have some common
+features.
+
+## Summary
+
+At the broadest level, there are three features to be added resulting from this RFC.
+
+* Linking of Works to Series
+* Linking of parent Works to child Works
+* Linking of child Works to parent Works
+
+## API change
+
+This requires a change to the API:
+* `id` is currently required in partOf objects, it needs to be optional.
+* a new field needs to be added to part and partOf objects: `subPart`
+
+## Preliminary Work
+
+We must ensure that the website will not attempt to render inappropriate
+hierarchies when we start adding new part/partOf relationships.
+
+## Staged development proposal
+
+1. All parent relationships described in this document to be displayed as series links. 
+   - This includes 773 relationships with identifiers, which will eventually link between Works.
+   - The subpart ($g or $v) is ignored
+2. Improve handling of identified Work->Work links for hierarchical display.
+3. Improve handling of asymmetric partOf links to link directly by id.
+4. Display the subPart if desired.
+
+This approach allows us to first deliver an improvement to the linking of
+as many records as possible, and then iterates over _how_ that linking
+can be improved for those records where we have more information.
+
+It also represents a cone of uncertainty.  It is clear that we want to link
+from Works to Series via a filtered search for the Series title.  The 
+correct presentation of links between parent and child works is less clear. 
+In particular, how best to handle asymmetric links.
+The value of presenting the subPart text to users is not clear.
+
+## Relevant MARC fields
+
+* [440 - Series Statement/Added Entry-Title](https://www.loc.gov/marc/bibliographic/bd440.html)
+* [490 - Series Statement](https://www.loc.gov/marc/bibliographic/bd490.html)
+* [773 - Host Item Entry](https://www.loc.gov/marc/bibliographic/bd773.html)
+* [774 - Constituent Unit Entry](https://www.loc.gov/marc/bibliographic/bd774.html)
+* [830 - Series Added Entry-Uniform Title](https://www.loc.gov/marc/bibliographic/bd830.html)
+
+
+## Common features
+
+### Most Precise Wins
+
+The 773 field is not restrictive about what it can point to, whereas the 4x0 and 830 
+fields explicitly refer to a Series.  Records with a 773 field that matches
+one of the series fields should ignore the 773 field and just present it
+as a Series link.
+
+If the title of a 773 field matches one of the Series fields, but it also
+contains a $w subfield, then 
+__Investigate where this happens in order to decide, before developing a solution__
+ * Find out if it ever happens
+ * Find out how many objects there are in each "series" where it does happen
+ * Ask C&R what it means
```

Comment:
Yes, I included this because it is logically possible, I am yet to investigate whether such a thing actually exists.  If it does, then getting it cleaned sounds like the most likely outcome.